staging/src/k8s.io/code-generator/cmd/informer-gen/generators/versioninterface.go

```diff
@@ -68,7 +68,7 @@ func (g *versionInterfaceGenerator) GenerateType(c *generator.Context, t *types.
 
 	sw.Do(versionTemplate, m)
 	for _, typeDef := range g.types {
-		tags, err := util.ParseClientGenTags(typeDef.SecondClosestCommentLines)
+		tags, err := util.ParseClientGenTags(append(typeDef.SecondClosestCommentLines, typeDef.CommentLines...))
```

Comment:
sigh, another one. compare https://github.com/kubernetes/gengo/pull/121

Reply:
Please add a comment to https://github.com/kubernetes/gengo/pull/121#issuecomment-411692391

Reply:
![bildschirmfoto 2018-08-23 um 16 57 16](https://user-images.githubusercontent.com/730123/44533350-a4fc5a00-a6f5-11e8-8559-b7cf37b7deeb.png)

And yes, I think we make this mistake at a couple of places. Worth a fix.

Reply:
Today I learned :)

Please use `append(append([]string{}, t.SecondClosestCommentLines...), t.CommentLines...)` to avoid landmines.

Reply:
maybe gengo based code is safe because everything is single-threaded.